tmpl/README.md.j2

```diff
@@ -54,8 +53,7 @@ cd <package-root-dir>
 pytest
 ```
 
-Note that neither pytest, nor pre-commit, nor any of the linters invoked by the pre-commit hooks will be available in the production environment, so make sure you have a development environment
-installed and activated. If you use the blueprint as is, pre-commit will not be triggered locally but only if you push to the main branch
+If you use the blueprint as is, pre-commit will not be triggered locally but only if you push to the main branch
```

Comment:
I'd replace "use the blueprint" by, e.g., "use the tools provided by the blueprint", since (in my understanding) the blueprint is only "used" to create (or update) a project.

Reply:
Yes this is better, thanks. I changed it.